tests/phpunit/CRM/Batch/Form/EntryTest.php

```diff
@@ -27,58 +27,61 @@
 use Civi\Api4\Campaign;
 
 /**
- *  Test CRM/Member/BAO Membership Log add , delete functions
```

Comment:
I think it's great to have a descriptor in a class docblock ... but it should describe the class it is in - not the one it was copied from :-)